docs/hazmat/primitives/symmetric-encryption.rst

```diff
@@ -469,6 +469,29 @@ Modes
 
         a secret message!
 
+.. class:: XTS(tweak)
+
+    .. versionadded:: 2.1
+
+    .. warning::
+
+        XTS mode is meant for disk encryption and should not be used in other
+        contexts.
+
+    .. note::
+
+        AES XTS keys are double length. This means that to do AES 128
+        encryption in XTS mode you need a 256-bit key. Similarly, AES 256
+        requires passing a 512-bit key. AES 192 is not supported in XTS mode.
+
+    XTS (XEX-based tweaked-codebook mode with ciphertext stealing) is a mode
+    of operation for the AES block cipher that is used for `disk encryption`_.
+
+    **This mode does not require padding.**
+
+    :param bytes tweak: The tweak is a 16 byte value typically derived from
+        something like the disk sector number.
```

Comment:
More precise description please.

Reply:
Do you have language you want to see? I'm not sure how to make this more precise since the details appear to be almost entirely dependent on specific implementation. We could mention that reuse of a tweak is not catastrophic in the same way that nonce reuse is I guess?

I've added a little language around that but better wording welcome!